Conclude Beginners Global Taiko Showdown 2020 (#4919)
ppy/osu-wiki: wiki/Tournaments/GTS/BGTS_2020/en.md

| ![][flag_VE] | **Venezuela** | **[RaFaReAcH](https://osu.ppy.sh/users/10491903)**, [Angiie12](https://osu.ppy.sh/users/7577264), [eblf2013](https://osu.ppy.sh/users/2419080), [Rushino](https://osu.ppy.sh/users/4605933) |
| ![][flag_VN] | **Vietnam** | **[Noname1999](https://osu.ppy.sh/users/11489158)**, [39BlueStars](https://osu.ppy.sh/users/12438394), [miyatohanasaki](https://osu.ppy.sh/users/13916687), [thuw\_103](https://osu.ppy.sh/users/16946200) |

## All Stars participants

> Both teams were formed by a vote held in the GTS Discord server. Participants with the most votes got to be the captain for each team, and the rest of the members were decided by rankings as stated in the [All Stars rules](#all-stars-rules).

Reviewer: ```suggestion
Both teams were formed by a vote held in the GTS Discord server. Participants with the most votes became the captains for each team, and the rest of the members were decided by rankings as stated in the [All Stars rules](#all-stars-rules).
```
